### ISSUE TITLE
Doesn't Want to load Search Results 

### ISSUE THREAD

**Issue by YSOYT** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#424]
### Bug alert 
Doesn't load Most Searches  / Searches that have a lot of info or link in them 
(Examples: most food recipes )

and doesn't load YouTube search results 
![image](https://user-images.githubusercontent.com/79786546/121983820-3b600580-cd92-11eb-9c2b-58154ac85305.png)

**OS** 
Windows (build 19043)

**Comment by sumeshir26** [comment]
Hi @YSOYT,
This used to work, but then the Google Assistant SDK changed the response to "Here is what I found"/"Here is a result from YouTube" which tricks the app into thinking that it is a text result.
To fix this, nothing can be changed from the app's end.
(Duplicate of #41, #38, #53, and #131)

**Comment by ratsuns** [comment]
is there anything we can do to get this to work?

**Comment by sumeshir26** [comment]
@ratsuns The only solution I can think of is clicking the 'G Search' suggestion to opening the query in the browser to view the full results.
I know this is not a real solution but this is the best I can think of.

**Comment by ratsuns** [comment]
I think somebody else mentioned adding in some code but after looking at the codebase It looks like that suggestion was already implemented. 

It has something to do with Google recognizing it as a text rather than an image search.

**Comment by obed9057** [comment]
Yeah because tehn google would have to update the api Sent from Mail for Windows 10 From: sumeshir26Sent: June 15, 2021 2:15 PMTo: Melvin-Abraham/Google-Assistant-Unofficial-Desktop-ClientCc: SubscribedSubject: Re: [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client] Doesn't Want to load Search Results ***@***.*** The only solution I can think of is clicking the 'G Search' suggestion to opening the query in the browser to view the full results.I know this is not a real solution but this is the best I can think of.—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.
